src/components/loader.js

```diff
@@ -24,11 +21,11 @@ module.exports.Component = registerComponent('loader', {
     var url = parseUrl(data.src);
     var mtlUrl = parseUrl(data.mtl);
     var format = data.format;
+
     if (model) { el.removeObject3D('mesh'); }
-    if (!url) {
-      warn('Model URL not provided');
-      return;
-    }
+
+    if (!url) { return console.warn('Model `src` was not specified'); }
```

Comment:
I'm just going to use `console` for critical warnings/errors for now so that dev/users can actually see them without doing `localStorage.logs = 1` which isn't documented very well. Have an issue to expose warnings/errors but hide log messages.